src/devices/cpu/z80/tmpz84c015.h

```diff
@@ -26,8 +26,11 @@ class tmpz84c015_device : public z80_device
 {
 public:
 	tmpz84c015_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t);
+	tmpz84c015_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t, address_map_constructor io_map);
 
```

Comment:
Please make the second constructor here `protected` – it should only be used for derived classes.